<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDefaults</class>
 <widget class="QWidget" name="PreferencesDefaults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Font for lists and tables</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <property name="margin">
       <number>5</number>
      </property>

      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="font"/>
      </item>

      <item>
       <widget class="QLabel" name="label_default1">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Font size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fontsize"/>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Animations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>5</number>
      </property>

      <item>
       <widget class="QLabel" name="label_help4">
        <property name="toolTip">
         <string extracomment="Help info 1"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>In some actions, e.g. when zooming the dive profile, the changing axis values are animated. Select the speed with which this animation should occur (maximum = 500):</string>
        </property>
       </widget>
      </item>

      <item>  
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
            <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="velocitySlider">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="velocitySpinBox">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_dark">
     <property name="title">
      <string>Dark theme text colours</string>
     </property>
     <layout class="QVBoxLayout" name="darkModeLayout_4">
      <property name="margin">
       <number>5</number>
      </property>

      <item>
       <widget class="QLabel" name="label_help_dark">
        <property name="toolTip">
         <string extracomment="Help info dark"/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>With some computers, when a dark theme is chosen, some of the blue text in Subsurface is not clearly visible. Select a colour that fits the current theme of your computer. For dark mode, select either Light Blue or Black (rendered white using a dark theme). The default is Medium Blue.  IMPORTANT: You need to restart Subsurface for this change to take effect.</string>
        </property>
       </widget>
      </item>

      <item>  
       <layout class="QHBoxLayout" name="darkmodeLayout_5">

         <item>
          <widget class="QRadioButton" name="mediumblue_text">
           <property name="text">
            <string>Medium Blue</string>
           </property>
          </widget>
         </item>

         <item>
          <widget class="QRadioButton" name="lightblue_text">
           <property name="text">
            <string>Light Blue</string>
           </property>
          </widget>
         </item>

         <item>
          <widget class="QRadioButton" name="black_text">
           <property name="text">
            <string>Black</string>
           </property>
          </widget>
         </item>

        </layout>
      </item>

      </layout>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>195</height>
      </size>
     </property>
    </spacer>
   </item>
      <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>195</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections>
  <connection>
   <sender>velocitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velocitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velocitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
